cmd/stop.go

```diff
@@ -23,12 +23,16 @@ func newStopCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	// stopCmd represents the stop command
 	var stopCmd = &cobra.Command{
 		Use:   "stop",
-		Short: "Stops the local Appsody docker container for your project",
-		Long: `Stop the local Appsody docker container for your project.
+		Short: "Stop the local running Appsody container.",
+		Long: `Stop the local running Appsody container for your project.
```

Comment:
Think some clarity is needed here, but not sure what! Either:
"Stop the local, running Appsody container for your project."  (ie, it's local and it's running)
or
"Stop the local-running Appsody container for your project."  (ie, it's running locally, as opposed to something running remotely)